src/proto/Tezos.proto

```diff
@@ -36,18 +36,16 @@ message Operation {
     // Note: Proto3 semantics require a zero value.
     ENDORSEMENT = 0;
 
-    REVEAL = 7;
-    TRANSACTION = 8;
-    ORIGINATION = 9;
-    DELEGATION = 10;
+    REVEAL = 107;
+    TRANSACTION = 108;
```

Comment:
why they changed this? does not make any sense

Reply:
I don't know.

Reply:
https://tezos.gitlab.io/mainnet/protocols/005_babylon.html#the-tags-of-all-manager-operations-are-shifted-by-100